dev/jackdaw/repl.clj

```diff
@@ -115,21 +114,6 @@
   given with EDN serdes and a partition count of one."
   (FakeTopicMetadata.))
 
-(defn topology->topic-metadata
```

Comment:
☝️ This function infers the Kafka topics needed by a topology. We can't simply delete it as this will break examples.

Reply:
Ah sorry I thought this ns was only used in the repl.
Can we just add the grapher as dependency to the examples?

Reply:
That is one option.

The namespace is referenced here:
https://github.com/FundingCircle/jackdaw/blob/master/examples/word-count/dev/user.clj#L32

Reply:
Ok thanks I'll fix the examples then